pkg/services/app/add.go

```diff
@@ -547,41 +550,28 @@ func (a *App) writeAppGoats(basePath string, name string, clusterName string, ma
 }
 
 func generateAppYaml(params AddParams, repo string) ([]byte, error) {
-	const appYamlTemplate = `---
-apiVersion: wego.weave.works/v1alpha1
-kind: Application
-metadata:
-  name: {{ .AppName }}
-  namespace: {{ .Namespace }}
-  labels:
-    weave-gitops.weave.works/app-identifier: {{ .AppHash }}
-spec:
-  path: {{ .AppPath }}
-  url: {{ .AppURL }}
-`
-	// Create app.yaml
-	t, err := template.New("appYaml").Parse(appYamlTemplate)
-	if err != nil {
-		return nil, errors.Wrap(err, "could not parse app yaml template")
+	gvk := wego.GroupVersion.WithKind(wego.ApplicationKind)
+	app := wego.Application{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       gvk.Kind,
+			APIVersion: gvk.GroupVersion().String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      params.Name,
+			Namespace: params.Namespace,
```

Comment:
We still need to have the appHash for the label so we dont add two of the same applications